apps/web/lib/integrations/calendar/services/Office365CalendarService.ts

```diff
@@ -35,7 +35,7 @@ export default class Office365CalendarService implements Calendar {
         ? `${event.destinationCalendar.externalId}/`
         : "";
 
-      const response = await fetch(`https://graph.microsoft.com/v1.0/me/calendar/${calendarId}events`, {
+      const response = await fetch(`https://graph.microsoft.com/v1.0/me/calendar/events`, {
```

Comment:
Fixed this route based on this doc
https://docs.microsoft.com/en-us/graph/api/user-post-events?view=graph-rest-1.0&tabs=http